PERFORMANCE.md

```diff
@@ -4,36 +4,48 @@
+	* safely >1 Mbit/s
+	* format still a WIP. To be continued...
 
 * details:
 	* cimbar has built-in compression using zstd. What's being measured here is bits over the wire, e.g. data after compression is applied.
-	* these numbers are using https://github.com/sz3/cfc, running with 4 CPU threads on a Qualcomm Snapdragon 625
-		* perhaps I will buy a new cell phone to inflate the benchmark numbers.
-	* the sender is the cimbar.org wasm implementation. An equivalent command line is `./cimbar_send /path/to/file -s`
+	* these numbers are using https://github.com/sz3/cfc, running with 4 CPU threads on a venerable Qualcomm Snapdragon 625
+		* more modern cell CPUs run the decoder more quickly, but it turns out that this does not benefit performance much: the camera is usually the bottleneck.
+	* the sender is the cimbar.org wasm implementation. An equivalent command line is `./cimbar_send /path/to/file`
 		* cimbar.org uses the `shakycam` option to allow the receiver to detect/discard "in between" frames as part of the scan step. This allows it to spend more processing time decoding real data.
 	* burst rate can be higher (or lower)
-		* to this end, lower ecc settings *can* provide better burst rates
-	* 4-color cimbar is currently preferred, and will give more consistent transfer speeds.
-	* 8-color cimbar should be considered a prototype within a prototype. It is considerably more sensitive to lighting conditions and color tints.
+		* to this end, lower ecc settings *can* provide better burst rates. I've aimed for a balance of performance and reliability.
+	* cimbar `mode B` is preferred, and should be the most reliable.
+	* The older `mode 4C` *may* give more consistent transfer speeds in certain scenarios, but is mostly included for backwards-compatibility reasons.
 
 * other notes:
-	* having better lighting in the frame often leads to better results -- this is why cimbar.org has a (mostly) white background. cfc uses android's auto-exposure, auto-focus, etc (it's a very simple app). Good ambient light -- or a white background -- can lead to more consitent quality frame capture.
+	* having better lighting in the frame often leads to better results -- this is why cimbar.org has a (mostly) white background. cfc uses android's auto-exposure, auto-focus, etc (it's a demo app). Good ambient light -- or a white background -- can lead to more consitent quality frame capture.
+		* screen brightness on the sender is good, but ambient light is better.
 	* because of the lighting/exposure question, landscape *may* be better than portrait.
-	* cfc currently has a low resolution, so the cimbar frame should take up as much of the display as possible (trust the guide brackets)
+	* the cimbar frame should take up as much of the display as possible (trust the guide brackets)
+		* the format is designed to decode at resolutions as low as 700x700, but performance may suffer.
```

Comment:
In running some experiments for the 5x5 configuration (and 8x8 as the control), I was able to shrink the encoded image incrementally down to 700x700 without seeing significant performance degradation. This was a loose goal of the initial 8x8, but I wasn't sure where the limit actually was.